app/models/user.rb

```diff
@@ -1,2 +1,11 @@
 class User < ApplicationRecord
+	has_many :friendsips
```

Comment:
we don't really have these models just yet.. should we include these relationships?